thoth/storages/graph/postgres.py

```diff
@@ -2299,6 +2315,9 @@ def sync_analysis_result(self, document: dict) -> None:
         os_name = document["result"]["operating-system"]["name"]
         os_version = document["result"]["operating-system"]["version_id"]
 
+        # Check if it comes from a User
+        is_external = document["metadata"]["arguments"]["thoth-package-extract"]["metadata"]["is_external"]
```

Comment:
To keep ability to sync older results and wait also for changes to be propagated to deployment, I would suggest:

```python
        is_external = document["metadata"]["arguments"]["thoth-package-extract"]["metadata"].get("is_external", True)
```

Reply:
Perfect, I will adjust it, Thanks :)